lib/common/environment.dart

```diff
@@ -142,6 +145,12 @@ class Environment {
 
   static String get cloudFlareImageUrlPrefix =>
       dotenv.env['CLOUD_FLARE_IMAGE_URL_PREFIX'] ?? '';
+
+  static get tokenWebviewTestnewPrefix =>
```

Comment:
I think we don't need to split into 2 env here, just 1 `TOKEN_WEBVIEW_PREFIX` is enough. Can you please update the Bitwarden also (just testnet, I will update staging and prd later).